### DISCUSSION TITLE
Make the docs page autofill the examples with the logged-in user's real token

### DISCUSSION THREAD

**Discussion by CONIGUERO** [Infisical/infisical#5705] [Feature Request]
In many of the docs pages there are examples for how to run many commands that require an infiscal API token. Currently it just shows `[token]` as a placeholder. It would be great if the user could login to the docs page and the token placeholder be replaeable with specific tokens that the user has.

**Comment by maidul98** [comment]
This is an awesome idea, however, we use https://mintlify.com/ for our docs an I'm unsure if they support this at the moment. I'll let them know so they can add to their feature list but thank you for the suggestion!
